yb-voyager/cmd/importData.go

```diff
@@ -591,10 +618,20 @@ func updateTargetConfInMigrationStatus() {
 
 func importData(importFileTasks []*ImportFileTask, errorPolicy importdata.ErrorPolicy) {
 
-	if (importerRole == TARGET_DB_IMPORTER_ROLE || importerRole == IMPORT_FILE_ROLE) && (tconf.EnableUpsert) {
-		if !utils.AskPrompt(color.RedString("WARNING: Ensure that tables on target YugabyteDB do not have secondary indexes. " +
-			"If a table has secondary indexes, setting --enable-upsert to true may lead to corruption of the indexes. Are you sure you want to proceed?")) {
-			utils.ErrExit("Aborting import.")
+	switch importerRole {
```

Comment:
let's create a separate function for this. perhaps `UpdateImportDataStartedinMetaDB` or something like that.